### PR TITLE
Bit imports babel

### PR DESCRIPTION
**Pull request by MiguelCastillo**

### CODE REVIEW

**Comment by MiguelCastillo** [comment]
@BrianGenisio this PR is for renaming 6to5 to babel...  I am having a hard time adjusting. Particularly that it is really easy for me to type bable in stead of babel :D

**Comment by BrianGenisio** [comment]
Need to update the `arrow-functions-context` example since `window` is no longer the context.

**Comment by MiguelCastillo** [comment]
- [ ] arrow-functions-context [href](http://learnharmony.org/#lessons/arrow-functions-context)
- [ ] string-methods [href](http://learnharmony.org/#es6discuss/string-methods)

**Comment by MiguelCastillo** [comment]
How about something like this for arrow function context:
``` javascript
var ctx = window;

// this === window
let printThis = () => {
  if (this !== ctx) {
    console.log("Context is not changed");
  }
}

printThis();
printThis.call(ctx);
printThis.apply(ctx);
printThis.bind(ctx)();
```

**Comment by BrianGenisio** [comment]
I think `ctx` doesn't need to be `window`.  `window` is too crazy big.

```js
var ctx = { foo: 'bar' }
```

**Comment by MiguelCastillo** [comment]
sure, that'll work too.  I am hoping you can give me a better message than `Context is not changed` :)

File: site/src/editor/editor.js [hunk 1]
```diff
@@ -30,13 +30,17 @@ class Editor {
...
-
     let moduleName = 'repl' + this.moduleId++;
-    let transpiled = this.compiler.transpile(code, moduleName);
-    eval(transpiled);
-    System.import(moduleName);
+
+    this.compiler
+      .transpile(code, moduleName)
+      .then(function transpileSuccess(transpiled) {
+        System.register(moduleName, [], function() { return eval(transpiled); });
```
Comment (MiguelCastillo) [inline site/src/editor/editor.js]: This is where I do the module registration

**Comment by MiguelCastillo** [comment]
Corrected issue with learnharmony not running in firefox and the rest of the klan. I am now using the pollyfill from babel, which includes a slightly modified version of regenerator.  So, we no longer need to load regenerator ourselves.

**Comment by MiguelCastillo** [comment]
@BrianGenisio http://learnharmony.org/#es6discuss/map-set does not seem to run in chrome and firefox.  I just get an empty application container and the example isn't loaded
screen shot 2015-03-05 at 10 45 00 pm

**Comment by MiguelCastillo** [comment]
@BrianGenisio all the items are ready for you to take a second pass.  EXCEPT for importing a module the way you need.  I will need to think about that problem a little bit because this one *might* require some changes in the module loader itself.  Do you think we can track this one as a separate item outside of this PR?

**Comment by BrianGenisio** [comment]
Sure.  Especially since it is only used (right now, at least) when I demo during my presentation... which I don't have lined up any time in the future... I'm sure I will again, but we have time to think about it separately.

**Comment by MiguelCastillo** [comment]
i want to log an issue for looking into alternatives for deployment to see if we can prevent checking in node_modules... but I think that should be done as a separate PR.

**Comment by BrianGenisio** [comment]
I'll look at it tonight.

**Comment by MiguelCastillo** [comment]
here is the module import functionality https://github.com/BrianGenisio/learnharmony/issues/12

**Comment by BrianGenisio** [comment]
Looks like we are still missing polyfills for new array methods.  `[].find()`, `[].findIndex()`, and `[].fill()`.  

[href](http://learnharmony.org/#es6discuss/string-methods)

**Comment by BrianGenisio** [comment]
RE: the Map-Set link, I had a typo.  It should be maps-sets.  http://learnharmony.org/#es6discuss/maps-sets

We still have a problem that Maps work but Sets do not.  The Babel polyfill doesn't seem to add them.  What about including this: https://github.com/WebReflection/es6-collections

**Comment by BrianGenisio** [comment]
We're getting there :)

**Comment by MiguelCastillo** [comment]
Cool, I will add es6-shim and try again :)

**Pull request merged**